xl/version.py

```diff
@@ -41,7 +41,7 @@ def get_current_revision(directory):
     try:
         return subprocess.check_output([
             'git', 'rev-parse', '--short=7', 'HEAD'
-        ]).strip()
+        ], stderr=-1).strip()
```

Comment:
Ah, this is subprocess.PIPE. The python documentation states "Do not use stderr=PIPE with this function as that can deadlock based on the child process error volume". And while the output *should* be small, I'd rather not risk it.

I don't mind suppressing the output, but there has to be a better way.